test/images/image-util.sh

```diff
@@ -56,7 +56,8 @@ build() {
 
     # Create a temporary directory for every architecture and copy the image content
     # and build the image from temporary directory
-    temp_dir=$(mktemp -d)
+    mkdir -p ${KUBE_ROOT}/_tmp
+    temp_dir=$(mktemp -d ${KUBE_ROOT}/_tmp/test-images-build.XXXXXX)
```

Comment:
@spiffxp : looks like we have a pattern already in our scripts

`kube::util::ensure-temp-dir` and then use `KUBE_TEMP` wanna use that?

https://cs.k8s.io/?q=%3A%3Aensure-temp-dir%7CKUBE_TEMP&i=nope&files=&repos=

Reply:
At least on my macOS system, that ends up making a tmpdir under `/var/folders`, which is what I'm trying to avoid.  I'm wary of messing with that function because while I know using `_tmp` works here, I haven't tested everything else that uses this.

Reply:
maybe add a trap to cleanup this temp dir?

otherwise this looks fine.

Reply:
ack, trap added